invenio_oaiserver/response.py

```diff
@@ -280,7 +280,13 @@ def header(parent, identifier, datestamp, sets=None, deleted=False):
 
 def getrecord(**kwargs):
     """Create OAI-PMH response for verb Identify."""
-    record_dumper = serializer(kwargs['metadataPrefix'])
+    metadataPrefix = (
+        kwargs.get('resumptionToken').get('metadataPrefix')
+        if kwargs.get('resumptionToken')
+        else kwargs['metadataPrefix']
+    )
+    record_dumper = serializer(metadataPrefix)
+
```

Comment:
Can use the one liner above, as the `GetRecord` will not have a `resumptionToken`